src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -39,6 +44,39 @@ def read_and_clean_excel_file(excelfile: str, sheetname: str | int = 0) -> pd.Da
     return read_df
 
 
+def read_and_clean_all_sheets_excelfile(excelfile: str) -> dict[str, pd.DataFrame]:
```

Comment:
how about renaming this to `read_and_clean_all_sheets()` without the `excelfile` which seems a bit redundant